src/clients/SpokePoolClient.ts

```diff
@@ -747,7 +757,7 @@ export class SpokePoolClient extends BaseAbstractClient {
         let deposit: DepositWithBlock;
 
         if (this.isV3DepositEvent(event)) {
-          deposit = { ...(rawDeposit as V3DepositWithBlock) };
+          deposit = { ...(rawDeposit as V3DepositWithBlock), originChainId: this.chainId };
```

Comment:
Not in v3 - we omitted it because the consumer of the event already knows what the chain ID is. It's fine as long as the consumer isn't forgetful, as we were here :facepalm: 

![v3](https://github.com/across-protocol/sdk-v2/assets/108695806/6079cdb2-0d28-4033-a5f3-b5c02353cc82)